File: app/src/main/java/com/qhy040404/libraryonetap/datamodel/SessionData.java
package com.qhy040404.libraryonetap.datamodel;

import com.google.gson.Gson;

import java.io.Serializable;

public class SessionData implements Serializable {
    Gson gson = new Gson();

    private static class GsonData {
        private boolean success;
        private String message;
        private String user_id;
    }

    public boolean isSuccess(String returnData) {
        GsonData gsonData = gson.fromJson(returnData, GsonData.class);
        return gsonData.success;
    }
}

File: app/src/main/java/com/qhy040404/libraryonetap/datamodel/SessionData.kt
package com.qhy040404.libraryonetap.datamodel

import com.qhy040404.libraryonetap.constant.GlobalManager.gson

object SessionData {
    fun isSuccess(returnData: String?): Boolean {
        return gson.fromJson(returnData, SessionDataClass::class.java).success
    }
}